supabase/functions/toggle-rsvp/index.ts

```
@@ -0,0 +1,94 @@
+        headers: { ...corsHeaders, "Content-Type": "application/json" },
+      });
+    }
+
+    // Rate Limiting Logic using Deno KV (Atomic Centralized Store)
+    const kv = await Deno.openKv();
+    const rateLimitKey = ["rate_limit", user.id, eventId];
+    const now = Date.now();
+    const lastRequest = await kv.get<number>(rateLimitKey);
+
+    if (lastRequest.value && now - lastRequest.value < RATE_LIMIT_WINDOW_MS) {
+      return new Response(JSON.stringify({ error: "Rate limit exceeded. Please wait before toggling again." }), {
```

ESLint

[error] Replace `JSON.stringify({·error:·"Rate·limit·exceeded.·Please·wait·before·toggling·again."·}),` with `⏎········JSON.stringify({·error:·"Rate·limit·exceeded.·Please·wait·before·toggling·again."·}),⏎·······`

(prettier/prettier)